tests/cases/tm2eth/scripts/test-service

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+: <<'END_COMMENT'
+* relay-interval = 20s
+* src-relay-optimize-interval = 30s
+* src-relay-optimize-count = 3
+* dst-relay-optimize-interval = 30s
+* dst-relay-optimize-count = 3
+END_COMMENT
+
+set -eux
+
+SCRIPT_DIR=$(cd $(dirname $0); pwd)
+source $SCRIPT_DIR/utils
+RLY_BIN=${SCRIPT_DIR}/../../../../build/yrly
+RLY="${RLY_BIN} --debug"
+
+if [ "USE_FAKELOST-$USE_FAKELOST" = "USE_FAKELOST-yes" ]; then
+    export DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_ibc1="10"
+    export DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER_ibc1="10"
+    export DEBUG_RELAYER_SHFU_WAIT_ibc0="20"
+fi
+
+PATH_NAME=mockapp-path
+TM_ADDRESS0=$(${RLY} tendermint keys show ibc0 testkey)
+
+expectUnrelayedCount $PATH_NAME "unrelayed-packets" "src" 0
+expectUnrelayedCount $PATH_NAME "unrelayed-packets" "dst" 0
+expectUnrelayedCount $PATH_NAME "unrelayed-acknowledgements" "src" 0
+expectUnrelayedCount $PATH_NAME "unrelayed-acknowledgements" "dst" 0
+
+sendPacketToEth $PATH_NAME
+waitUnrelayedCount $PATH_NAME "unrelayed-packets" "dst" 1 300
+
+docker exec tendermint-chain0-mock sh -c "simd --home /root/data/ibc0 tx --keyring-backend=test --from ${TM_ADDRESS0} --chain-id ibc0 mockapp send mockapp channel-0 'mock packet data' --yes"
+waitUnrelayedCount $PATH_NAME "unrelayed-packets" "src" 1 30
+
+date; echo "Server has started. Both chains already have a message to relay which will be ready after optimize-interval (30s)."
```

Comment:
@dai1975 nit? I think we should move this line AFTER the server has really started.